dev/ProgressBar/ProgressBar.xaml

```diff
@@ -65,6 +70,22 @@
                                 <VisualState x:Name="Normal" />
                                 <VisualState x:Name="Determinate" />
                                 <VisualState x:Name="Updating" />
+                                <VisualState x:Name="UpdatingError">
+                                    <Storyboard>
+                                        <Storyboard>
+                                            <ColorAnimation
+                                            Storyboard.TargetName="DeterminateProgressBarIndicator"
+                                            Storyboard.TargetProperty="(Shape.Fill).(SolidColorBrush.Color)"
+                                            To="{Binding Source={ThemeResource SystemControlErrorTextForegroundBrush},Path=Color}"
+                                            Duration="0:0:0.0"/>
+                                            <ColorAnimation
+                                            Storyboard.TargetName="ProgressBarRoot"
+                                            Storyboard.TargetProperty="(Border.Background).(SolidColorBrush.Color)"
+                                            To="{StaticResource SystemControlErrorBackgroundColor}"
+                                            Duration="0:0:0.0"/>
```

Comment:
Is a setter more appropriate than a 0 duration story board?

Reply:
Quite frankly, I wasn't able to get a setter to that property working. I've tried different variations of brackets and they always crashed.